.rubocop.yml

```diff
@@ -2,7 +2,7 @@ inherit_from:
   - conf/rubocop.yml
 
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.5
```

Comment:
Should be bump this (and the minimum Ruby version in the gemspec) to Ruby 2.6 since Ruby 2.5 is EOL now?